ICD10PCS/readme.md

```diff
@@ -5,13 +5,51 @@ Prerequisites:
 - Working directory ICD10PCS.
 
 1. Run create_source_tables.sql
-2. Download the latest file from https://www.cms.gov/Medicare/Coding/ICD10/YYYY-ICD-10-PCS-and-GEMs.html (e.g. https://www.cms.gov/Medicare/Coding/ICD10/2019-ICD-10-PCS-and-GEMs.html), 
+2. Download the latest file from https://www.cms.gov/medicare/coding-billing/icd-10-codes,
 file name YYYY-ICD-10-PCS-Order-File.zip (e.g. 2019-ICD-10-PCS-Order-File.zip) listed as "ICD-10-PCS Order File (Long and Abbreviated Titles)"
 3. Exctract icd10pcs_order_YYYY.txt and rename to icd10pcs.txt
 4. Run in devv5 (with fresh vocabulary date and version): SELECT sources.load_input_tables('ICD10PCS',TO_DATE('20180101','YYYYMMDD'),'ICD10PCS 20180101'); (pVocabularyDate=YYYY-1)
-5. Run load_stage.sql
-6. Run generic_update: devv5.GenericUpdate();
 
-Directory snomed_hierarchy contains unrefined version of tools to build automated relationship to SNOMED concepts currently stored in concept_relationship_manual. These scripts are early version and require a thorough refactoring before being reused.
+6. Run FastRecreateSchema:
+```sql
+   SELECT devv5.FastRecreateSchema(main_schema_name=>'devv5', include_concept_ancestor=> true, include_deprecated_rels=> true, include_synonyms=> true);
+```
+7. Run load_stage.sql.
+8. Run generic_update:
+  ```sql
+SELECT admin_pack.VirtualLogIn('dev_mkhitrun','MKh_388646467');
```

Comment:
Please remove it and change the password

Reply:
Done